frontend/src/views/admin/matching/import-export.tsx

```diff
@@ -0,0 +1,442 @@
+import React from "react";
+import FileSaver from "file-saver";
+import { matchingDataSelector } from "./actions";
+import { useSelector } from "react-redux";
+import { Modal, Button, Row, Form, Col } from "react-bootstrap";
+import { DataFormat } from "../../../libs/import-export";
+import { Match, AppointmentGuaranteeStatus } from "./types";
+import { upsertMatch, batchUpsertGuarantees, upsertNote } from "./actions";
+import { useThunkDispatch } from "../../../libs/thunk-dispatch";
+
+export function ExportMatchingDataButton() {
+    const matchingData = useSelector(matchingDataSelector);
+
+    function onClick() {
+        const blob = new Blob([JSON.stringify(matchingData)], {
+            type: "text/plain;charset=utf-8",
+        });
+        const date = new Date();
+        FileSaver.saveAs(
+            blob,
+            "matching_data-" +
+                date.getFullYear() +
+                "-" +
+                date.getMonth().toString().padStart(2, "0") +
+                "-" +
+                date.getDate().toString().padStart(2, "0") +
+                "-" +
+                date.getHours().toString().padStart(2, "0") +
+                "-" +
+                date.getMinutes().toString().padStart(2, "0") +
+                "-" +
+                date.getSeconds().toString().padStart(2, "0") +
+                ".json"
```

Comment:
Use the `toLocaleString` date formatting function instead. Also, no string concatenation. I don't think minutes and hous are helpful. Just format for the current date and the user can add more if they want.